File: Zad3/app/src/main/java/com/example/bmizad2/ui/recipes/RecipesViewModel.java
package com.example.bmizad2.ui.recipes;

import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.bmizad2.R;


public class RecipesViewModel extends ViewModel {
    private final MutableLiveData<String> mText;

    public RecipesViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: Zad4/app/src/main/java/com/example/bmizad2/ui/recipes/RecipesViewModel.kt
package com.example.bmizad2.ui.recipes

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class RecipesViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()

    init {
        mText.value = ""
    }

    val text: LiveData<String>
        get() = mText
}